KimFom.ShiftLogger/ShiftLoggerConsole/ShiftLoggerConsole/Services/ApiConnectionService.cs

```diff
@@ -0,0 +1,88 @@
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using Microsoft.Extensions.Configuration;
+using ShiftLoggerConsole.Dtos;
+using ShiftLoggerConsole.Models;
+
+namespace ShiftLoggerConsole.Services;
+
+public class ApiConnectionService : IApiConnectionService
+{
+    private readonly HttpClient _httpClient;
+
+    public ApiConnectionService(HttpClient httpClient, IConfiguration configuration)
+    {
+        _httpClient = httpClient;
+        _httpClient.DefaultRequestHeaders.Accept.Clear();
+        _httpClient.BaseAddress = new Uri(configuration.GetSection("BaseUrl").Value!);
+        _httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+    }
+
+    public async Task<List<Shift>?> GetAllShifts()
+    {
+        List<Shift>? shiftList = new();
+        
```

Comment:
One last thing before we approve it. Could you please show a more meaningful "duration" to the user? I like to pick on these things because it's always good to practice dealing with date, time and durations. Also I just noticed the message is "press any enter" 😊

![image](https://user-images.githubusercontent.com/108711476/211945525-d8facc83-dd75-4555-8789-78ab23f81c75.png)

Reply:
Press any enter 😂😂😂 
Okay I'll do that later today

Reply:
Should I also include minutes?

Reply:
Yeah but seconds aren't necessary I don't think 😁

Reply:
Alrighty I'll work on that tonight